exercises/practice/two-fer/test/tests.chpl

```diff
@@ -0,0 +1,17 @@
+use UnitTest;
+
+use helloWorld;
```

Comment:
use thing that might be good to highlight is that chapel modules cannot use `kebab-case`, but I guess automatically converting it to e.g. `camelCase` should not be too difficult?

Reply:
actually, I would say it's more chapel style to start module names with a capital letter, sot that should probably be `TwoFer` (this is my fault, I was sloppy with the hello world example :) )

Reply:
Oh! Let me fix that.